compiler/modules/front/const.bal

```diff
@@ -503,7 +503,7 @@ function foldedUnaryConstExpr(SimpleConst value, t:UniformTypeBitSet basicType,
 }
 
 function foldVarRefExpr(FoldContext cx, t:SemType? expectedType, s:VarRefExpr expr) returns s:Expr|FoldError {
-    s:FLOAT_ZERO|t:Value? constValue = check cx.lookupConst(expr.prefix, expr.varName);
+    s:FLOAT_ZERO|t:Value? constValue = check cx.lookupConst(expr.prefix, expr.varName, expr.startPos);
```

Comment:
Shouldn't VarRefExpr have a namePos field, which we should use here? (Since we are not representing parenthesized expressions explicitly, it may be different.)

Reply:
added `namePos` field to used it here